fedimint-api/src/db/mod.rs

```diff
@@ -281,6 +380,13 @@ impl<'a> DatabaseTransaction<'a> {
         }
     }
 
+    pub async fn notify_key<K>(&mut self, key: &K)
```

Comment:
maybe this should be renamed to add_notification_key? That might help address the confusion I had earlier

Reply:
I was thinking the same thing.

Reply:
done